app/src/main/java/org/gdg/frisbee/android/arrow/ArrowTaggedActivity.java

```diff
@@ -120,7 +119,10 @@ public void onResult(AppStateManager.StateResult stateResult) {
 
                     @Override
                     public void success(final Directory directory, final retrofit.client.Response response) {
-                        String[] orgas = taggedOrganizers.split("\\|");
+                        if (orgas != null) {
```

Comment:
This is a good fix but I think it ignores the newly added items. Imagine
you are in GDG Summit and just tagged a person, you would want to see the
person you've just tagged.
I actually don't know what is happenning there. If you say it is ok, it is
ok.

On Mon, Jul 27, 2015, 4:05 PM Marcus Gabilheri <notifications@github.com>
wrote:

> In
> app/src/main/java/org/gdg/frisbee/android/arrow/ArrowTaggedActivity.java
> <https://github.com/gdg-x/frisbee/pull/421#discussion_r35532699>:
>
> > @@ -120,7 +119,10 @@ public void onResult(AppStateManager.StateResult stateResult) {
> >
> >                      @Override
> >                      public void success(final Directory directory, final retrofit.client.Response response) {
> > -                        String[] orgas = taggedOrganizers.split("\\|");
> > +                        if (orgas != null) {
>
> So. If you are in the tagged organizes viewing the list and hit Home
> button and then come back to the app the list just gets populated again
> having a duplicate of each item. It just keep growing and growing. That
> prevents the list from duplicating items once it has already been populated.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/421/files#r35532699>.
>